### PR TITLE
xorg_sgml_doctools, new recipe

### PR DESCRIPTION
**Pull request by Begasus**

### CODE REVIEW

File: x11-doc/xorg_sgml_doctools/xorg_sgml_doctools-1.11.recipe [hunk 1]
```diff
@@ -0,0 +1,50 @@
+SUMMARY="A common set of SGML entities and XML/CSS style sheets"
+DESCRIPTION="This package provides a common set of SGML entities and XML/CSS \
+style sheets used in building/formatting the documentation provided in other\
+X.Org packages.   It's typically only needed by people building from \
+source who want to produce formatted documentation from their builds, \
+or those who have installed the HTML version of the documentation, \
+which refers to the included common xorg.css stylesheet."
+HOMEPAGE="http://xorg.freedesktop.org/releases/individual/proto/"
```
Comment (fbrosson) [inline x11-doc/xorg_sgml_doctools/xorg_sgml_doctools-1.11.recipe]:
```diff
-HOMEPAGE="http://xorg.freedesktop.org/releases/individual/proto/"
+HOMEPAGE="https://xorg.freedesktop.org/releases/individual/proto/"
```

File: x11-doc/xorg_sgml_doctools/xorg_sgml_doctools-1.11.recipe [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+COPYRIGHT="2010, Oracle and/or its affiliates
```
Comment (fbrosson) [inline x11-doc/xorg_sgml_doctools/xorg_sgml_doctools-1.11.recipe]:
```diff
-COPYRIGHT="2010, Oracle and/or its affiliates
+COPYRIGHT="2010 Oracle and/or its affiliates
```

File: x11-doc/xorg_sgml_doctools/xorg_sgml_doctools-1.11.recipe [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+	2005 Red Hat, Inc."
+LICENSE="MIT"
+REVISION="1"
+SOURCE_URI="https://github.com/freedesktop/xorg-sgml-doctools/archive/xorg-sgml-doctools-$portVersion.tar.gz"
```
Comment (alaviss) [inline x11-doc/xorg_sgml_doctools/xorg_sgml_doctools-1.11.recipe]: I think it's better to fetch the tarball from it's home :)

https://www.x.org/releases/individual/doc/xorg-sgml-doctools-1.11.tar.bz2

File: x11-doc/xorg_sgml_doctools/xorg_sgml_doctools-1.11.recipe [hunk 4]
```diff
@@ -0,0 +1,50 @@
+SUMMARY="A common set of SGML entities and XML/CSS style sheets"
+DESCRIPTION="This package provides a common set of SGML entities and XML/CSS \
+style sheets used in building/formatting the documentation provided in other\
+X.Org packages.   It's typically only needed by people building from \
```
Comment (alaviss) [inline x11-doc/xorg_sgml_doctools/xorg_sgml_doctools-1.11.recipe]:
```diff
-style sheets used in building/formatting the documentation provided in other\
-X.Org packages.   It's typically only needed by people building from \
+style sheets used in building/formatting the documentation provided in other \
+X.Org packages. It's typically only needed by people building from \
```

File: x11-doc/xorg_sgml_doctools/xorg_sgml_doctools-1.11.recipe (hunk 1 above)
Comment (alaviss) [reply x11-doc/xorg_sgml_doctools/xorg_sgml_doctools-1.11.recipe]: This should be the real homepage: https://www.x.org/releases/individual/doc/

File: x11-doc/xorg_sgml_doctools/xorg_sgml_doctools-1.11.recipe [hunk 5]
```diff
@@ -0,0 +1,50 @@
...
+	cmd:autoconf
+	cmd:gcc
+	cmd:ld
+	cmd:make
+	cmd:pkg_config
+	"
+
+BUILD()
+{
+	autoreconf -vfi
```
Comment (alaviss) [inline x11-doc/xorg_sgml_doctools/xorg_sgml_doctools-1.11.recipe]: Is this required?

File: x11-doc/xorg_sgml_doctools/xorg_sgml_doctools-1.11.recipe [hunk 6]
```diff
@@ -0,0 +1,50 @@
...
+	runConfigure ./configure
+}
+
+INSTALL()
+{
+	make install
+	fixPkgconfig
```
Comment (alaviss) [inline x11-doc/xorg_sgml_doctools/xorg_sgml_doctools-1.11.recipe]: If the `.pc` files are in `dataDir` (it have to if we want to classify this as an `any` package), then we can remove `fixPkgconfig` from the recipe.

File: x11-doc/xorg_sgml_doctools/xorg_sgml_doctools-1.11.recipe (hunk 1 above)
Comment (fbrosson) [reply x11-doc/xorg_sgml_doctools/xorg_sgml_doctools-1.11.recipe]: We could also use both :)
```
HOMEPAGE="https://www.x.org/releases/individual/doc/
	https://xorg.freedesktop.org/releases/individual/proto/"
```
But I guess I'm ok with any single line `HOMEPAGE` too.

Comment (Begasus) [reply x11-doc/xorg_sgml_doctools/xorg_sgml_doctools-1.11.recipe]: hmm ... I'm guessing */doc should be used (as */proto links to the proto related downloads)
Or just "https://www.xorg.com"?

**Pull request merged**